doc/specs/vulkan/chapters/copies.txt

```diff
@@ -436,6 +436,24 @@ ifdef::VK_KHR_sampler_ycbcr_conversion[]
 endif::VK_KHR_sampler_ycbcr_conversion[]
   * [[VUID-vkCmdCopyImage-srcImage-00136]]
     The sample count of pname:srcImage and pname:dstImage must: match
+  * The slink:VkImageCopy::pname:srcSubresource::pname:mipLevel members of
+    the elements of the pname:pRegions array must: each be less than the
```

Comment:
There's actually mandated style for this now, see https://www.khronos.org/registry/vulkan/specs/1.0/styleguide.html#writing-arrays

Reply:
@oddhack Good to know. I will revise it.

Reply:
@krOoze would you be able to get the style changes done on Thursday? Otherwise it will be a few weeks before we can accept PRs, as we have some planned downtime and won't be updating github for 3 weeks or so.

Reply:
@oddhack You mean today? I guess I should be able to unless I get stuck.

Reply:
@oddhack Restyled.